### ISSUE TITLE
Web version is crashed!

### ISSUE THREAD

**Issue by dihar** [fkling/astexplorer#325]
Hello! Now there is unhandled exception on https://astexplorer.net/ website.
### Steps to reproduce:
1) Open site
2) Focus left column
3) Select all (cmd + A)
4) Something paste

### Result. 
Interface disappears, in console appears errors:
```
Uncaught Error: React is running in production mode, but dead code elimination has not been applied. Read how to correctly configure React for production: https://fb.me/react-perf-use-the-production-build
    at <anonymous>:21:35
```
```
vendor-fdcf8e110acf76aa650b-16.js:80 Error: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=object%20with%20keys%20%7Blabel%2C%20keyword%2C%20beforeExpr%2C%20startsExpr%2C%20rightAssociative%2C%20isLoop%2C%20isAssign%2C%20prefix%2C%20postfix%2C%20binop%2C%20updateContext%7D&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings. 
    at invariant (vendor-fdcf8e110acf76aa650b-16.js:1483)
    at A (vendor-fdcf8e110acf76aa650b-16.js:79)
    at rg (vendor-fdcf8e110acf76aa650b-16.js:80)
    at s (vendor-fdcf8e110acf76aa650b-16.js:80)
    at h (vendor-fdcf8e110acf76aa650b-16.js:80)
    at vendor-fdcf8e110acf76aa650b-16.js:80
    at Eg (vendor-fdcf8e110acf76aa650b-16.js:80)
    at Q (vendor-fdcf8e110acf76aa650b-16.js:80)
    at Kg (vendor-fdcf8e110acf76aa650b-16.js:80)
    at th (vendor-fdcf8e110acf76aa650b-16.js:80)
```
```TypeError: Invalid attempt to destructure non-iterable instance
    at vendor-fdcf8e110acf76aa650b-16.js:1499
    at t.value (app-263ea36250d2440aaa24-16.js:1155)
    at Wg (vendor-fdcf8e110acf76aa650b-16.js:80)
    at Xg (vendor-fdcf8e110acf76aa650b-16.js:80)
    at Xh (vendor-fdcf8e110acf76aa650b-16.js:80)
    at Rh (vendor-fdcf8e110acf76aa650b-16.js:80)
    at Uh (vendor-fdcf8e110acf76aa650b-16.js:80)
    at Sh (vendor-fdcf8e110acf76aa650b-16.js:80)
    at wh (vendor-fdcf8e110acf76aa650b-16.js:80)
    at kg (vendor-fdcf8e110acf76aa650b-16.js:80)
```

Thanks!

**Comment by fkling** [comment]
Which parser/transform have you selected? I can see the first error (though that shouldn't impact the UI), but not the second one.

**Comment by fkling** [comment]
OK, I can repro this in **Firefox** and **Safari**, but not in Chrome.

**Comment by graingert** [comment]
I can repro in Chrome:

```
Uncaught Error: React is running in production mode, but dead code elimination has not been applied. Read how to correctly configure React for production: https://fb.me/react-perf-use-the-production-build
    at <anonymous>:21:35
vendor-fdcf8e110acf76aa650b-16.js:80 Error: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=object%20with%20keys%20%7Blabel%2C%20keyword%2C%20beforeExpr%2C%20startsExpr%2C%20rightAssociative%2C%20isLoop%2C%20isAssign%2C%20prefix%2C%20postfix%2C%20binop%2C%20updateContext%7D&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings. 
    at invariant (vendor-fdcf8e110acf76aa650b-16.js:1483)
    at A (vendor-fdcf8e110acf76aa650b-16.js:79)
    at rg (vendor-fdcf8e110acf76aa650b-16.js:80)
    at s (vendor-fdcf8e110acf76aa650b-16.js:80)
    at h (vendor-fdcf8e110acf76aa650b-16.js:80)
    at vendor-fdcf8e110acf76aa650b-16.js:80
    at Eg (vendor-fdcf8e110acf76aa650b-16.js:80)
    at Q (vendor-fdcf8e110acf76aa650b-16.js:80)
    at Kg (vendor-fdcf8e110acf76aa650b-16.js:80)
    at th (vendor-fdcf8e110acf76aa650b-16.js:80)
Tg @ vendor-fdcf8e110acf76aa650b-16.js:80
vendor-fdcf8e110acf76aa650b-16.js:80 TypeError: Invalid attempt to destructure non-iterable instance
    at vendor-fdcf8e110acf76aa650b-16.js:1499
    at t.value (app-263ea36250d2440aaa24-16.js:1155)
    at Wg (vendor-fdcf8e110acf76aa650b-16.js:80)
    at Xg (vendor-fdcf8e110acf76aa650b-16.js:80)
    at Xh (vendor-fdcf8e110acf76aa650b-16.js:80)
    at Rh (vendor-fdcf8e110acf76aa650b-16.js:80)
    at Uh (vendor-fdcf8e110acf76aa650b-16.js:80)
    at Sh (vendor-fdcf8e110acf76aa650b-16.js:80)
    at wh (vendor-fdcf8e110acf76aa650b-16.js:80)
    at kg (vendor-fdcf8e110acf76aa650b-16.js:80)
Tg @ vendor-fdcf8e110acf76aa650b-16.js:80
vendor-fdcf8e110acf76aa650b-16.js:80 Uncaught Error: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=object%20with%20keys%20%7Blabel%2C%20keyword%2C%20beforeExpr%2C%20startsExpr%2C%20rightAssociative%2C%20isLoop%2C%20isAssign%2C%20prefix%2C%20postfix%2C%20binop%2C%20updateContext%7D&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings. 
    at invariant (vendor-fdcf8e110acf76aa650b-16.js:1483)
    at A (vendor-fdcf8e110acf76aa650b-16.js:79)
    at rg (vendor-fdcf8e110acf76aa650b-16.js:80)
    at s (vendor-fdcf8e110acf76aa650b-16.js:80)
    at h (vendor-fdcf8e110acf76aa650b-16.js:80)
    at vendor-fdcf8e110acf76aa650b-16.js:80
    at Eg (vendor-fdcf8e110acf76aa650b-16.js:80)
    at Q (vendor-fdcf8e110acf76aa650b-16.js:80)
    at Kg (vendor-fdcf8e110acf76aa650b-16.js:80)
    at th (vendor-fdcf8e110acf76aa650b-16.js:80)
```

**Comment by graingert** [comment]
Specifically when I click the last blank line (20) on `acorn-to-esprima`

**Comment by fkling** [comment]
@graingert But not for other parsers?

**Comment by fkling** [comment]
Since `acorn-to-esprima` isn't directly available from the menu anyway, I updated the code to default to a parser that is available from the menu (ea89f10692f8830942bcdb27457b376c1ad87dbd).

It's more of a workaround than an actual fix but I haven't figured out yet what the problem is.

Please let me know if you encounter this problem again. Thank you!
